include/ck/tensor_operation/gpu/grid/normalization/gridwise_normalization_splitk_1st.hpp

```diff
@@ -78,14 +78,14 @@ struct GridwiseNormalizationSplitK1st
     static constexpr auto ThreadBufferNumber = Number<KThreadSliceSize / XSrcVectorSize>{};
 
     __device__ static int
-    GetKPerThread(int kRaw, int kGridSize, int block_k_cluster_id, int thread_k_cluster_id)
+    GetKPerThread(int k, int kRaw, int kGridSize, int block_k_cluster_id, int thread_k_cluster_id)
     {
         bool is_rightmost_block = block_k_cluster_id == kGridSize - 1;
 
         if(is_rightmost_block)
         {
-            int left_kPerBlock = math::integer_divide_ceil(kRaw, kGridSize);
-            int kPerBlock      = kRaw % kGridSize == 0 ? left_kPerBlock : kRaw % left_kPerBlock;
+            int left_kPerBlock = math::integer_divide_ceil(k, kGridSize);
+            int kPerBlock      = kRaw - left_kPerBlock * (kGridSize - 1);
```

Comment:
Using better name to replace `left_kPerBlock`, `kPerBlock`  like `kSizePerBlock`,  `kSizeRightmostBlock`

Reply:
b41c98f411da01febf80a9094e3a441240123c0d